### PR TITLE
Add a missing test case to the `CommandTrackerFlowTest` [KVL-1104]

### PR DESCRIPTION
**Pull request by hubert-da**
CHANGELOG_BEGIN
CHANGELOG_END

### Pull Request Checklist

- [x] Read and understand the [contribution guidelines](https://github.com/digital-asset/daml/blob/main/CONTRIBUTING.md)
- [x] Include appropriate tests
- [x] Set a descriptive title and thorough description
- [x] Add a reference to the [issue this PR will solve](https://github.com/digital-asset/daml/issues), if appropriate
- [ ] Include changelog additions in one or more commit message bodies between the `CHANGELOG_BEGIN` and `CHANGELOG_END` tags
- [ ] Normal production system change, include purpose of change in description

NOTE: CI is not automatically run on non-members pull-requests for security
reasons. The reviewer will have to comment with `/AzurePipelines run` to
trigger the build.

### CODE REVIEW

File: ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala [hunk 1]
```diff
@@ -494,12 +494,13 @@ class CommandTrackerFlowTest
     }
 
     "a completion without the submission id arrives" should {
+
```
Comment (hubert-da) [inline ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala:497]: Just to make it consistent with other test cases

**Review by SamirTalwar-DA — APPROVED** [review]

File: ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala [hunk 2]
```diff
@@ -581,7 +582,29 @@ class CommandTrackerFlowTest
...
+        val Handle(submissions, results, _, completionStreamMock) =
+          runCommandTrackingFlow(allSubmissionsSuccessful)
+
+        results.request(2)
+
+        submissions.sendNext(newSubmission("submissionId", commandId))
+        submissions.sendNext(newSubmission("anotherSubmissionId", commandId))
+
+        completionStreamMock.send(successfulCompletion("submissionId", commandId))
+        completionStreamMock.send(successfulCompletion("anotherSubmissionId", commandId))
```
Comment (andreaslochbihler-da) [inline ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala:597]: What about the case where we get a rejection and a successful completion?

Comment (hubert-da) [reply ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala:597]: Not sure if it's worth exploring more here, as there are tests for failures in the stream.

Comment (hubert-da) [reply ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala:597]: i.e. failures are not being handled differently than success responses

Comment (andreaslochbihler-da) [reply ledger/ledger-api-client/src/test/suite/scala/com/digitalasset/ledger/client/services/commands/CommandTrackerFlowTest.scala:597]: OK. You know the implementation better and you're the one who has to be happy with the test coverage in the end.

**Pull request merged**